ncm-metaconfig/src/main/metaconfig/haproxy/pan/schema.pan

```diff
@@ -21,6 +21,11 @@ type haproxy_service_global_config = {
     'daemon' : string = ''
     'pidfile' : string = '/var/run/haproxy.pid'
     'chroot' : string = '/var/lib/haproxy'
+
+    'ssl-default-bind-ciphers' ? string_non_whitespace
```

Comment:
shouldnt this be an array?

Reply:
@kwaegema it looks like `   ssl-default-server-ciphers ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE...`
So an array will not do afaik.

Reply:
Or in other words: it should be, but the TT file will need updating.

Reply:
yes. I will make an update to this PR to fix all this.

Reply:
Awesome thanks!